.prettierrc

```diff
@@ -6,5 +6,5 @@
   "tabWidth": 2,
   "trailingComma": "es5",
   "bracketSpacing": true,
-  "endOfLine": "lf"
+  "endOfLine": "auto"
```

Comment:
I don't think we should change this.

Why did you have to do that?

Reply:
tried to work with a windows set up and was doing some debugging, reverted change back